### DISCUSSION TITLE
Conda package

### DISCUSSION THREAD

**Discussion by johnwason** [hungpham2511/toppra#175] [General]
I created a conda package with the fixes in #173 for use with my project:

https://anaconda.org/pyri-project/toppra/files

The feedstock:

https://github.com/pyri-project-conda/toppra-feedstock

It would probably be good to submit it to conda-forge, but I will leave that to the owner.

**Comment by hungpham2511** [comment]
Thanks for the suggestion. I totally miss this for some reason. Not much of a conda user myself, so I will probably need to do some readings to see how this works.

Just curious, why do you personally use conda and not pip?

**Comment by johnwason** [comment]
The project we are working on (https://github.com/pyri-project/pyri-core) is too complicated to deploy on pip. Conda allows for better deployment of components and dependencies in C++ and other languages.
